acceptance-tests/src/test/java/tech/pegasys/web3signer/dsl/signer/runner/Web3SignerRunner.java

```diff
@@ -259,4 +278,47 @@ public Path createJksCertFile(final TlsCertificateDefinition caTrustStore) {
   protected SignerConfiguration getSignerConfig() {
     return signerConfig;
   }
+
+  // Assumes database already exists, just enforces the tables exist as necessary.
+  @SuppressWarnings("UnusedVariable")
+  private void createSlashingDatabase(
+      final String url, final String username, final String password) {
+    final HikariDataSource dataSource = new HikariDataSource();
+    dataSource.setUsername(username);
+    dataSource.setPassword(password);
+    dataSource.setJdbcUrl(url);
+
+    try {
+      final Connection conn = dataSource.getConnection();
+      //      final Statement stmt = conn.createStatement();
```

Comment:
looks like leftover testing code? can this be removed

Reply:
removed